contrib/Trivy.gitlab-ci.yml

```diff
@@ -0,0 +1,27 @@
+Trivy_container_scanning:
+  stage: test
+  image:
+    name: alpine
+  variables:
+    # Override the GIT_STRATEGY variable in your `.gitlab-ci.yml` file and set it to `fetch` if you want to provide a `clair-whitelist.yml`
+    # file. See https://docs.gitlab.com/ee/user/application_security/container_scanning/index.html#overriding-the-container-scanning-template
+    # for details
+    GIT_STRATEGY: none
+    IMAGE: "$CI_REGISTRY_IMAGE:$CI_COMMIT_SHA"
+  allow_failure: true
+  before_script:
+    - apk add --no-cache curl docker-cli
+    - docker login -u "$CI_REGISTRY_USER" -p "$CI_REGISTRY_PASSWORD" $CI_REGISTRY
+    - curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sh -s -- -b /usr/local/bin
+  script:
+    - trivy --exit-code 0 --cache-dir $CI_PROJECT_DIR/.trivycache/ --no-progress --format template --template "@contrib/gitlab.tpl" -o ${CI_PROJECT_DIR}/gl-container-scanning-report.json $IMAGE
```

Comment:
`contrib/gitlab.tpl` is missing now while the install script was improved.

Reply:
@knqyf263 Could you help me find out what is wrong? It is confirmed that template exists and the command arguments look correct: https://gitlab.com/tnir/trivy-ci-test/-/jobs/412246842

Reply:
I'm sorry the install script fetches the latest version of Trivy, but the latest version doesn't support a file template yet. I'll release it soon.

Reply:
I've released v0.4.3. Would you try it again?

Reply:
Thanks. It works now! :smile: https://gitlab.com/tnir/trivy-ci-test/-/jobs/414121133

Script will be updated when other matters are addressed.